client/dashboard/sites/deployments/dataviews/fields.tsx

```diff
@@ -139,7 +142,18 @@ export function useDeploymentFields( {
 					return item.created_on;
 				},
 				render: ( { item } ) => {
-					return <TimeSince timestamp={ item.created_on } />;
+					return (
+						<Button
+							variant="link"
+							onClick={ () => onOpenLogs( item ) }
```

Comment:
I agree with the others, let's open the modal as the main interaction. For the date, let's just show a tooltip with the full timestamp. I added more context to [the issue](https://linear.app/a8c/issue/DOTDASH-373/add-modal-for-deployment-details-and-logs#comment-beafaeba).

Reply:
Regarding the tooltips, I'll tackle them in a separate issue, as they are not related to the logs.
I've created DOTDASH-494